03-client-api/00-overview.md

```diff
@@ -34,6 +37,8 @@ To avoid running into issues and make the most out of using a Grakn client, keep
 
 **Keep the number of operations per transaction minimal**. Although it is technically possible to commit a write transaction once after many operations, it is not recommended. To avoid lengthy rollbacks, running out of memory and conflicting operations, it is best to keep the number of queries per transaction minimal, ideally to one query per transaction.
 
+**Take advantage of asynchronous queries where possible.** This will cut down on network round-trip time and increase your throughput where possible. All queries can safely be made asynchronous, as they will automatically wait for previous queries to finishe on the server-side.
```

Comment:
To remain consistent, we use the present tense where it makes sense. so `This cuts down` over `This will cut down`.

finish**~~e~~**